MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -7069,6 +7072,17 @@ private void uidkey()
             Registers.AX = 1;
         }
 
+        /// <summary>
+        ///     Create a new key record
+        ///
+        ///     Signature: int nkyrec (char *uid);
+        /// </summary>
+        private void nkyrec()
+        {
+            // no key support for now, so everybody has the key
```

Comment:
Let's add a debug here that a new key record is being created and write out the value passed in. It'll help if this stubbed out version doesn't work for a module, well at least have the debug that the api is being called.

Reply:
Debug added